it/src/test/scala/com/test/raphtory/WriteValue.scala

```diff
@@ -11,4 +13,10 @@ case class WriteValue() extends GenericReduction {
       .step(vertex => vertex.setState("testing", 1))
       .reducedView(PropertyMergeStrategy.sum[Int])
 
+  override def tabularise(graph: ReducedGraphPerspective): Table =
+    graph
+      .step { vertex =>
+        vertex.setState("vertexname", vertex.name())
```

Comment:
will change to "name" - as a side point if we were ever to com back, calling "name" in select should just call the function on the vertex I think.